server/src/controllers/AuthController.js

```diff
@@ -65,27 +69,26 @@ module.exports = {
           }
         );
       }
+      const jwtUser = {
+        _id: user._id,
+        username: user.username,
+        email: user.email,
+        password: user.password,
```

Comment:
We are decoding the JWT on the browser, which would imply that the hashed password would be available. Maybe we can remove the password field here.